Functions/Admin/Backup-RsEncryptionKey.ps1

```diff
@@ -4,71 +4,134 @@
 
-    .PARAMETER SqlServerInstance (optional)
-        Specify the name of the SQL Server Reporting Services Instance.
-
-    .PARAMETER SqlServerVersion (optional)
-        Specify the version of the SQL Server Reporting Services Instance. 13 for SQL Server 2016, 12 for SQL Server 2014, 11 for SQL Server 2012
-
-    .PARAMETER Password
-        Specify the password to be used for backing up the encryption key. This password will be required when restoring the encryption key.
+	    .PARAMETER ReportServerInstance
+            Specify the name of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ReportServerVersion
+            Specify the version of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ComputerName
+            The Report Server to target.
+            Use the "Connect-RsReportServer" function to set/update a default value.
         
-    .PARAMETER KeyPath
-        Specify the path to where the encryption key should be stored. 
+        .PARAMETER Credential
+            The credentials with which to connect to the Report Server.
+            Use the "Connect-RsReportServer" function to set/update a default value.
 
-    .EXAMPLE
-        Backup-RSEncryptionKey -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
-        Description
-        -----------
-        This command will back up the encryption key against default instance from SQL Server 2016 Reporting Services
-    
-    .EXAMPLE
-        Backup-RSEncryptionKey -SqlServerInstance 'SQL2012' -SqlServerVersion '11' -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
-        Description
-        -----------
-        This command will back up the encryption key against named instance (SQL2012) from SQL Server 2012 Reporting Services 
+	    .PARAMETER Password
+	        Specify the password to be used for backing up the encryption key. This password will be required when restoring the encryption key.
+	        
+	    .PARAMETER KeyPath
+	        Specify the path to where the encryption key should be stored. 
+
+	    .EXAMPLE
+	        Backup-RSEncryptionKey -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
+	        Description
+	        -----------
+	        This command will back up the encryption key against default instance from SQL Server 2016 Reporting Services
+	    
+	    .EXAMPLE
+	        Backup-RSEncryptionKey -ReportServerInstance 'SQL2012' -ReportServerVersion '11' -Password 'Enter Your Password' -KeyPath 'C:\ReportingServices\Default.snk'
+	        Description
+	        -----------
+	        This command will back up the encryption key against named instance (SQL2012) from SQL Server 2012 Reporting Services
+	
+		.NOTES
+			Author:      ???
```

Comment:
Hm, certainly a point. Whatever benefit a select few have (quite possibly only me, since I'm so used to this system) would probably be outweighed by the dead weight it represents to everyone else.
I'd still prefer to keep at least part of the header section though (Author and Editors, maybe roll it together into Contributors). Credits make people feel better and I don't see where that would escalate into bloat.